recipes/dlib/win_boost.patch

```diff
@@ -0,0 +1,44 @@
+--- dlib/add_python_module
++++ dlib/add_python_module
+@@ -34,37 +34,16 @@
+ get_filename_component(PYTHON_PATH ${PYTHON_PATH} PATH)
+ set(CMAKE_PREFIX_PATH ${PYTHON_PATH})
+ 
+-
+-# To avoid dll hell, always link everything statically when compiling in
+-# visual studio.  This way, the resulting library won't depend on a bunch
+-# of other dll files and can be safely copied to someone elese's computer
+-# and expected to run.
+-if (MSVC)
```

Comment:
Yeah I totally understand! Hence why I just made a patch here that just ripped that out. I'm not sure you should change it to be honest given the complicated build setup required for dlib.